test/support/spec.rb

```diff
@@ -173,6 +173,19 @@ def min_long
   -((2**(8 * 8 - 2)) * 2)
 end
 
+def ruby_exe(code, args: nil)
+  Tempfile.create('ruby_exe.rb') do |file|
+    file.write(code)
+    file.rewind
+
+    output = `bin/natalie #{file.path} #{args}`
```

Comment:
This should check for the existence of `ENV['NAT_BINARY']` and use that if available:
```suggestion
    binary = ENV['NAT_BINARY'] || 'bin/natalie'
    output = `#{binary} #{file.path} #{args}`
```